Makefile

```diff
@@ -14,6 +14,7 @@ clean-build:
 	rm -fr pip-wheel-metadata
 	find . -name '*.egg-info' -exec rm -fr {} +
 	find . -name '*.egg' -exec rm -fr {} +
+	find . -type d -name __pycache__ -exec rm -rv {} +
```

Comment:
I've added this to also delete the __pycache__ folders, as it is handy before pushing packages to ipfs so the CLI does not complain.